### ISSUE TITLE
Completely destroyed my VS 2022 installation

### ISSUE THREAD

**Issue by alexdresko** [StefanKert/BuildVision#122]
I cannot load VS without getting a ton of errors after installing this plugin. I will probably have to reset VS to defaults in order to get any work done today. I do not have time to troubleshoot this, but I figured it was worth creating an issue so others might know of the danger.

**Comment by jgo-bvb** [comment]
Probably #120 @alexdresko ? Just uninstall the extension and you should be fine

**Comment by StefanKert** [comment]
Hi @alexdresko,

thank you very much for reaching out and sorry for the inconvenience caused. As @jgo-bvb posted, I do hope that uninstalling the extension helped resolving this issue. As I am not sure yet if this is a duplicate of #120 I'll keep this issue open.

If you have got a ActivityLog of the crash I can do further checks @alexdresko

**Comment by StefanKert** [comment]
Closing this issue since currently it is a bit hard to follow along with the different issues / states. If you still see issues with the latest 4.0 please create another one. Thank you very much :)
